pkgs/development/python-modules/pygerber/default.nix

```diff
@@ -0,0 +1,91 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+
+  # build-system
+  poetry-core,
+
+  # dependencies
+  numpy,
+  click,
+  pillow,
+  pydantic,
+  pyparsing,
+  typing-extensions,
+  tzlocal,
+
+  # optional dependencies
+  pygls,
+  lsprotocol,
+  drawsvg,
+  pygments,
+  shapely,
+
+}:
+
+let
+  pillow_10_4 = pillow.overrideAttrs (prev: {
```

Comment:
does it make sense to supply some variables for the `pillow` dependency as well, or is it likely to be at 10.4 forever?

Its personal of course, but I'd suggest not having the version in the variable name